about.css

```diff
@@ -0,0 +1,1149 @@
+  font-size: 16px;
+  color: aliceblue;
+}
+
+.mainProgram-wrapper {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  height: 300px;
+  width: 100%;
+  font-size: 20px;
+  color: aliceblue;
+  margin-top: 200px;
+}
+
+.mainwrapper-list {
+  background-color: hsla(220, 3%, 39%, 0.489);
+  border: 1px;
+  margin: 4px 15px 4px 15px;
+  height: 140px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  padding: 1rem 0.5rem;
+  list-style: none;
+  align-items: center;
+}
+
+.card-title h2 {
+  font-size: 12px;
+  color: #ec5242;
+  width: 100px;
+  font-weight: 600;
+  align-items: center;
+  display: flex;
+  justify-content: center;
+}
+
+.paragraph-wrapper {
+  padding-top: 25px;
+  align-items: center;
+  font-size: 11px;
+  width: 170px;
+  height: 140px;
+}
+
+.wholeProgram {
+  display: flex;
+  justify-content: center;
+  font-size: 15px;
+  height: 50px;
+  margin: 3rem;
+  margin-top: -88px;
+  align-items: center;
+  position: relative;
+  border: 1px;
+  background-color: #ec5242;
+}
+
+.wholeProgram a {
+  color: #d3d3d3;
+  text-decoration: none;
+}
+
+.wholeProgram1 {
+  display: none;
+}
+
+.mainwrappe ul:hover {
+  border: 2px solid gray;
+}
+
+/* feature */
+.feature-speaker {
+  margin: 2rem;
+  display: grid;
+  margin-top: 5rem;
+}
+
+.single-feature-speker {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 1rem;
+  grid-template-columns: 1fr 1fr;
+  columns: 3rem;
+  width: 327px;
+  height: 200px;
+  margin-top: 2rem;
+}
+
+.image {
+  position: relative;
+}
+
+.speaker-description {
+  margin-left: 2rem;
+}
+
+.speaker-name {
+  padding-bottom: 0.5rem;
+  font-size: 0.9rem;
+}
+
+.speaker-details {
+  color: #ec5242;
+  font-size: 0.8rem;
+  font-style: italic;
+}
+
+.border-line1 {
+  display: none;
+}
+
+.border-line {
+  border-bottom: 1px solid;
+  width: 40px;
+  height: 1px;
+  background-color: #969595;
+  margin: 4px;
+}
+
+.speaker-background {
+  color: #272a31;
+  font-size: 0.8rem;
+}
+
+.bg-img {
+  position: absolute;
+  width: 70px;
+  z-index: -1;
+  top: -12%;
+  left: -20%;
+}
+
+.avatar {
+  display: block;
+  width: 130px;
+  height: 160px;
+}
+
+.FeauredSpeakers {
+  display: flex;
+  justify-content: center;
+  margin-top: 6rem;
+}
+
+hr {
+  width: 100px;
+  color: #e53d3d;
+  height: 7px;
+  margin-left: 150px;
+}
+
+.hidde {
+  display: none;
+}
+
+/* feature-speaker end  */
+
+/* partner section */
+
+#partner {
+  background-color: #515356;
+  padding: 2rem;
+}
+
+.partner-heading {
+  text-align: center;
+  color: wheat;
+}
+
+.partners-logos-container {
+  margin: 1rem 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  margin-top: 2rem;
+}
+
+.partners-logo {
+  width: 100px;
+}
+
+/* .partner section end  */
+
+/* footer section */
+#footer {
+  padding: 2rem;
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+}
```

Comment:
- [ ] As you can see on the design the about page footer has a dark background. Please kindly change the about page footer background color to the dark one 😉